cms/server/admin/templates/announcements.html

```diff
@@ -49,7 +48,28 @@ <h2 id="title_announcements" class="toggling_on">Announcements</h2>
           <div class="notification_admin_owner">
             By: {{ msg.admin.name if msg.admin is not none else "<unknown>" }}
           </div>
-
+          <a href="javascript:void(0);" onclick="utils.question_reply_toggle(event, this);">Edit</a>
```

Comment:
Would it be possible to have the subject and text in the edit prompt prefilled to the current values?